MediaBrowser.Model/Dlna/StreamBuilder.cs

```diff
@@ -1041,19 +1050,23 @@ private void BuildStreamVideoItem(
 
             var appliedAudioConditions = options.Profile.CodecProfiles
                 .Where(i => i.Type == CodecType.VideoAudio &&
-                    i.ContainsAnyCodec(audioCodecs, container) &&
+                    i.ContainsAnyCodec([.. audioCodecs], container) &&
                     i.ApplyConditions.All(applyCondition => ConditionProcessor.IsVideoAudioConditionSatisfied(applyCondition, audioChannels, inputAudioBitrate, inputAudioSampleRate, inputAudioBitDepth, audioProfile, isSecondaryAudio)))
                 // Reverse codec profiles for backward compatibility - first codec profile has higher priority
                 .Reverse();
 
-            foreach (var codecProfile in appliedAudioConditions)
+            var playlistAudioCodecs = playlistItem.AudioCodecs;
+            if (playlistAudioCodecs is not null)
             {
```

Comment:
Remove this check - `appliedAudioConditions` will be empty if there's no audio codec.